artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/TransportConfiguration.java

```diff
@@ -239,6 +239,17 @@ public boolean isSameParams(TransportConfiguration that) {
       return true;
    }
 
+   public boolean isSameTarget(TransportConfiguration otherConfig) {
+      if (this.getFactoryClassName().contains("Netty") && otherConfig.getFactoryClassName().contains("Netty")) {
+         return Objects.equals(this.params.get("host"), otherConfig.params.get("host")) &&
```

Comment:
does dns resolution come into play for the comparison? if a host is resolvable under two ips, would we try and cluster twice?